lcci/01.09.String Rotation/Solution.swift

```diff
@@ -0,0 +1,5 @@
+class Solution {
+    func isFlippedString(_ s1: String, _ s2: String) -> Bool {
+        return s1.count == s2.count && (s1 + s1).contains(s2)
```

Comment:
Hi @klever34 , In Swift code implementation, we need to check whether both string s1 and string s2 are empty. If they are, the answer is true.

```suggestion
        return (s1.isEmpty && s2.isEmpty) || (s1.count == s2.count && (s1 + s1).contains(s2))
```